engine/src/main/java/org/hibernate/validator/cfg/propertyholder/ContainerElementConstraintMappingContext.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.cfg.propertyholder;
+
+/**
+ * Constraint mapping creational context representing a type argument of a property, parameter or method return value
+ * with a generic (return) type. Allows to place constraints on that type argument, mark it as cascadable and to
```

Comment:
This is far more restricted in the case of a property holder (e.g. no methods for instance)